projects/canopy/src/lib/forms/date/date-field.component.html

```diff
@@ -10,7 +10,7 @@
   >
     <div class="lg-date-field__date">
       <label lg-label [for]="dateId">Day</label>
-      <lg-input-field>
+      <lg-input-field lgMarginBottom="none">
```

Comment:
I'm wondering if maybe the input shouldn't have margin by default? 🤔  Not specific to this PR but something to consider 🤔  cc @thatguynamedandy

Reply:
I always assumed that the input field needed margin because more often than not, you'd be adding them to a form stacked on top of each other, so having to add the correct margin to each one would be cumbersome and prone to error. But that's just been my assumption, rather than really considering it. What's the rationale for removing it?

Reply:
@owensgit is right with the intention, but there is also a downside. The hope was that the majority of time it would be easier as the default margin is in place. Links a bit to this https://github.com/Legal-and-General/canopy/issues/70

Reply:
Yes, agree but on the other hand it's very easy to use the margin directive. Otherwise when you have an input not followed by another input you have to override it.